File: code/cfg/NameCfg.cs
using System;
namespace cfg{
public class NameCfg  {
public readonly System.Collections.Generic.Dictionary<int, string> names = new System.Collections.Generic.Dictionary<int, string>();
public NameCfg(CSVStream fs) {
while(!fs.IsSectionEnd()) {
this.names[fs.GetInt()] = fs.GetString();
}
}
}
}

File: code/cfg/NameCfg.java
package cfg;
public class NameCfg  {
public final java.util.Map<Integer, String> names = new java.util.HashMap<Integer, String>();
public NameCfg(CSVStream fs) {
while(!fs.isSectionEnd()) {
this.names.put(fs.getInt(), fs.getString());
}
}
}